docs/how-to/service-dependencies.md

```diff
@@ -1,11 +1,297 @@
 # How to manage service dependencies
 
-Pebble takes service dependencies into account when starting and stopping services. When Pebble starts a service, it also starts the services which that service depends on (configured with `required`). Conversely, when stopping a service, Pebble also stops services which depend on that service.
+If you are using Pebble to manage your services, chances are, you've got more than one service to manage.
+
+And when orchestrating more services, things can and will get trickier, because more often than not, those services depend on each other to function together, and managing dependencies is no mean feat.
+
+Let's have a look at a concrete example of service dependencies.
+
+## My web app
+
+Imagine we are running a web application, which has:
```

Comment:
For how-tos I think the language should be more matter-of-fact.
https://diataxis.fr/how-to-guides/#the-language-of-how-to-guides

I suggest rewording this section to something like this.

To demonstrate the instructions in this guide, we are using a simple web
application with the following setup:

- A database listening on port 3306
- A backend server listening on port 8081, which talks to the database
- A frontend server listening on port 8080, which talks to the backend

The relationship between the frontend server, backend server and database can be simplified as:

`frontend (8080) -> backend (8081) -> database (3306)`

Reply:
I merged this suggestion with Ben's.